cristin-person/src/test/java/no/unit/nva/cristin/person/create/CreateCristinPersonHandlerTest.java

```diff
@@ -326,14 +327,16 @@ void shouldAllowEmptyListOfEmploymentsWhenUserActingAsThemselvesTryToCreateTheir
 
     @Test
     void shouldLogClientSpecificIdentifiersWhenDoingAuthorizedRequests() throws IOException {
-        final var standardOut = System.out;
```

Comment:
I think you need to keep this because it is assigned to something else further down. We need to keep the original stream before tampering with it, and then set it back when done. Maybe there is a better way, i dont know

Reply:
System.out is like a constant and is always open....

```
            try (var printStream = new PrintStream(outputStreamCaptor)) {
                System.setOut(printStream);
                var response = sendQueryWhileMockingIdentifiersUsedForLogging(dummyPerson());
                assertEquals(HTTP_CREATED, response.getStatusCode());
                System.setOut(System.out);
            }

```